datasets/cmip6-tas.data.mdx

```diff
@@ -26,7 +31,7 @@ infoDescription: |
 layers:
   - id: combined_CMIP6_daily_GISS-E2-1-G_tas_kerchunk_DEMO
     stacCol: combined_CMIP6_daily_GISS-E2-1-G_tas_kerchunk_DEMO
-    name: CMIP6 Daily GISS-E2-1-G Near-Surface Air Temperature (demo subset)
+    name: CMIP6 Daily GISS-E2-1-G Near-Surface Air Temperature (demo subset; 1950-2014)
     type: zarr
     tileApiEndpoint: 'https://prod-titiler-xarray.delta-backend.com/tilejson.json'
```

Comment:
Depending on timeline for titiler-xarray deployment we'll need to update these endpoints.